account_invoice_sale_order_warehouse/models/account_invoice.py

```diff
@@ -0,0 +1,21 @@
+# Copyright 2020 Quartile Limited
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class AccountInvoice(models.Model):
+    _inherit = "account.invoice"
+
+    sale_order_warehouse_id = fields.Many2one(
+        "stock.warehouse",
+        string="Sale Order Warehouse",
+        compute="_compute_sale_order_warehouse_id",
+    )
```

Comment:
Is non-stored filed OK for the customer (no sorting/grouping/filtering by the field)?

Reply:
I think it is ok since the main focus of the field is for the customer's portal page.